sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -1766,7 +1766,10 @@ class MapboxMap :
     return nativeMap.setViewAnnotationPositionsUpdateListener(listener)
   }
 
-  private fun checkNativeMap(methodName: String) {
+  private fun checkNativeMap(methodName: String, checkMainThread: Boolean = true) {
```

Comment:
Suppose the flag `checkMainThread` should be false for QRF?

Reply:
from what I see - it is